pytest_fixtures/core/contenthosts.py

```diff
@@ -268,8 +277,10 @@ def sat_upgrade_chost():
 def custom_host(request):
     """A rhel content host that passes custom host config through request.param"""
     deploy_args = request.param
-    # if 'deploy_rhel_version' is not set, let's default to RHEL 8
-    deploy_args['deploy_rhel_version'] = deploy_args.get('deploy_rhel_version', '8')
+    # if 'deploy_rhel_version' is not set, let's default to what's in server.yaml
+    deploy_args['deploy_rhel_version'] = deploy_args.get(
+        'deploy_rhel_version', settings.server.version.rhel_version
```

Comment:
Why server version? Why not version from `conf/content_host.yaml`?

Reply:
I guess this is not obvious from the description but this host becomes a Satellite so I opt to use the Satellite os version from the settings file and not content host.

Reply:
Since this custom_host fixture can be whatever you want let's do your change here and default to content_host.yaml and in my test we'll keep the server.yaml